src/test/rustdoc/unindent.rs

```diff
@@ -0,0 +1,27 @@
+#![feature(no_core)]
+
+#![crate_name = "foo"]
+#![no_std]
+#![no_core]
```

Comment:
Any reason to make this no_std and no_core?

Reply:
Woups, it was just to have less logs when running my tests. I removed them.